coveo-stew/README_MULTIPLE_LIBRARIES.md

```diff
@@ -0,0 +1,117 @@
+# Considerations for multi-projects repositories 
+
+Follow this guide to configure your repository to work with multiple projects.
+
+
+## Repository structure
+
+To combine multiple projects in a repository, you need to respect the following:
+
+1. At a defined root, such as the root of your repository or a folder within, create a new `pyproject.toml` file. It will be the dev environment; we will come back to this soon.
+2. Python libraries should then exist within that folder's hierarchy, in subfolders.
```

Comment:
Adding the support for sub-subfolders, is it in our plans?

I like it when we can regroup all the Python projects inside a `projects` or a `src` folder instead of the repo root. It fits better in repos that contain IaC code for example.

Reply:
That's exactly what the current line tries to convey! 😆 

"Within that folder's hierarchy, in subfolders" meaning, it can be sub-sub-sub-sub folders if you want.

Reply:
Rephrased:

> 2. Python libraries should then exist within that folder's hierarchy, each of them in a dedicated folder. The dedicated folders don't need to be in the root.

Reply:
You have a weird fetish with `'s`. :laughing: It should be `folder hierarchy`.